test/test_kfac.py

```diff
@@ -9,11 +10,11 @@
 from torch.nn import Module, MSELoss, Parameter
 
 from curvlinops.examples.utils import report_nonclose
-from curvlinops.ggn import GGNLinearOperator
 from curvlinops.gradient_moments import EFLinearOperator
 from curvlinops.kfac import KFACLinearOperator
 
 
+@mark.parametrize("separate_weight_and_bias", [True, False], ids=["separate", "joint"])
```

Comment:
The `ids` could a bit more verbose here, e.g. `ids=["separate_bias", "joint_bias"]`.

Reply:
Incorporated.